dapp/src/utility/proposal.utility.ts

```diff
@@ -250,3 +268,52 @@ export const checkProposalResult = (record: ProposalTableData) => {
 	}
 	return false;
 };
+
+/**
+ * Opt ASA to given account
+ * @param from : Account address that wants to opt asset
+ * @param assetID : Asset ID
+ * @param lsig : Logic Signature
+ * @param webMode : webmode instace to execute transaction
+ */
+export const optInASAToAccount = async (from: string, assetID: number, lsig: LogicSigAccount, webMode: any) => {
```

Comment:
I think it's the other way around. We opt in the account to a specific asa. Lets rename it